public/index.html

```diff
@@ -0,0 +1,493 @@
+              <div class="single-dot"></div>
+              <div class="single-dot"></div>
+            </div>
+          </li>
+          <li class="column-item">
+            <div class="info">
+              <h3>PREMIUM SUPPORT</h3>
+              <p>Eliefend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+            <span class="icon-apple sm-circular-eclipse-border"></span>
+            <div class="animated-dots">
+            </div>
+          </li>
+        </ul>
+        <img src="assets/iphoneFlat.png" alt="phone" class="why-choose-img">
+        <div class="arrow-position-wrapper">
+          <span class="drawn-arrow-blk">&nbsp;</span>
+        </div>
+        <ul class="column-two-list">
+          <li class="column-item">
+            <span class="icon-flag sm-circular-eclipse-border"></span>
+            <div class="info">
+              <h3>MULTIPURPOSE LAYOUT</h3>
+              <p>Eliefend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+            <div class="animated-dots">
+              <div class="single-dot"></div>
+              <div class="single-dot"></div>
+              <div class="single-dot"></div>
+            </div>
+          </li>
+          <li class="column-item">
+            <span class="icon-bubble sm-circular-eclipse-border"></span>
+            <div class="info">
+              <h3>LIVE CHAT SUPPORT</h3>
+              <p>Eliefend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+            <div class="animated-dots">
+              <div class="single-dot"></div>
+              <div class="single-dot"></div>
+              <div class="single-dot"></div>
+            </div>
+          </li>
+          <li class="column-item">
+            <span class="icon-mug sm-circular-eclipse-border"></span>
+            <div class="info">
+              <h3>AND MANY MORE</h3>
+              <p>Eliefend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+            <div class="animated-dots">
+            </div>
+          </li>
+        </ul>
+      </div>
+    <div class="app-review-wrapper fifth-skew">
+      <div class="arrow-position-wrapper">
+        <span class="sm-semicircular-eclipse-border"><span class="icon-quotes"></span></span>
+        <h2 id="reviews-nav" class="center-align">App Reviews</h2>
+        <h4>Read What Our Happy Customers Are Saying</h4>
+        <span class="drawn-arrow-wh">&nbsp;</span>
+      </div>
+      <div class="review-gallery sixth-skew">
+        <div>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</p>
+          <figure class="profile">
+            <span class="circle">
```

Comment:
I really try to avoid using overly simplified names. Using long names is NOT a bad idea. If the names adds value to the understanding of what it is without having prior knowledge of the design.